docs/orchestrating-elastic-stack-applications/elasticsearch/autoscaling.asciidoc

```diff
@@ -0,0 +1,197 @@
+:parent_page_id: elasticsearch-specification
+:page_id: autoscaling
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Elasticsearch autoscaling
+
+experimental[]
+
+ECK can leverage the autoscaling API introduced in Elasticsearch 7.11 to adjust automatically the number of Pods and the allocated resources in a tier. Currently, autoscaling is supported for Elasticsearch data tiers and machine learning nodes.
+
+[float]
+[id="{p}-enable"]
+== Enable autoscaling
+
+To enable autoscaling on an Elasticsearch cluster, you need to define one or more autoscaling policies. Each autoscaling policy applies to one or mode `NodeSets` which share the same set of roles specified in the `node.roles` setting in the Elasticsearch configuration.
+
+[float]
+[id="{p}-{page_id}-policies"]
+=== Define autoscaling policies
+
+Autoscaling policies can be defined in the `elasticsearch.alpha.elastic.co/autoscaling-spec` annotation of the Elasticsearch resource. Each autoscaling policy must have the following fields:
+
+* `name` is a unique name used to identify the autoscaling policy.
+* `roles` contains a set of node roles, unique across all the autoscaling policies, used to identify the `NodeSets` to which this policy applies. At least one `NodeSet` with the exact same set of roles must exist in the Elasticsearch resource specification.
```

Comment:
Nit: we haven't used backquotes around NodeSet in other [doc pages](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-orchestration.html#k8s-nodesets), so let's stay consistent?